dev-python/flask-babelex/flask-babelex-0.9.4.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit distutils-r1
+
+MY_PN="Flask-BabelEx"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Adds i18n/l10n support to Flask applications"
+HOMEPAGE="https://github.com/mrjoes/flask-babelex https://pypi.org/project/Flask-BabelEx/"
+SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+distutils_enable_sphinx docs
```

Comment:
Please read https://dev.gentoo.org/~mgorny/python-guide/distutils.html#building-documentation-via-sphinx and correct your mistakes.

Reply:
Do you mean that entire block:
doc? (
              	dev-python/sphinx[${PYTHON_USEDEP}]
                dev-python/flask-sphinx-themes[${PYTHON_USEDEP}]
        )

can be just removed since there is no need to explicitly depend on it??

Reply:
Yes, and src_compile, and src_install but you need to pass the dep to d_e_s.